command/oauth/cmd.go

```diff
@@ -438,11 +467,12 @@ type oauth struct {
 	CallbackPath        string
 	terminalRedirect    string
 	browser             string
+	authParams          map[string]string
```

Comment:
You should use [url.Values](https://pkg.go.dev/net/url#Values) or at what is the same `map[string][]string`